src/signup.js

```diff
@@ -0,0 +1,110 @@
+const emailContainer=document.querySelector('#email-input-container')
+const emailInput = emailContainer.querySelector('#email')
+const emailMsg = emailContainer.querySelector('#email-msg')
+const nameContainer=document.querySelector('#name-input-container')
+const nameInput = nameContainer.querySelector('#name')
+const nameMsg = nameContainer.querySelector('#name-msg')
```

Comment:
P2:
해당 변수명만 봤을때는 요소의 타입을 알기가 어렵습니다. 에러 메시지처럼 생각되는 변수명이니 
조금더 친절한 이름이면 좋을 것 같습니다.

```suggestion
const nameMsgEl = nameContainer.querySelector('#name-msg');
const nameMsgContainer = nameContainer.querySelector('#name-msg')
```